theme/src/resources/templates/header.hbs

```diff
@@ -751,7 +751,7 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 		const onNavScroll = function() {
 			const header = document.getElementById('header');
 
-			if (window.scrollY >= 64) {
+			if (window.scrollY >= 56) {
```

Comment:
This value seems like an arbitrary number.  Can we set it to a descriptive variable so we know what it is for?